en/Plugins/Getting started/Use Svelte in your plugin.md

```diff
@@ -38,27 +38,30 @@ To build a Svelte application, you need to install the dependencies and configur
    "inlineSourceMap": true,
    ```
 
-4. In `esbuild.config.mjs`, add the following imports to the top of the file:
+4. Add `.svelte` files to the `include` list in `tsconfig.json` so the bundler picks them up.
+
+   ```json
+   "include": [ "**/*.ts", "**/*.svelte" ]
+   ``` 
+
+5. In `esbuild.config.mjs`, add the following imports to the top of the file:
 
    ```js
    import esbuildSvelte from "esbuild-svelte";
    import sveltePreprocess from "svelte-preprocess";
```

Comment:
This should be changed to `import { sveltePreprocess } from 'svelte-preprocess';` as the current way it is, is a deprecated symbol.

It's shown here in the first code block below this [header](https://github.com/sveltejs/svelte-preprocess?tab=readme-ov-file#what-is-it)